functests/0_config/config.go

```diff
@@ -37,6 +40,22 @@ import (
 
 var _ = Describe("[performance][config] Performance configuration", func() {
 
+	It("Should run performance profile pod on a master node", func() {
+		pod, err := pods.GetPerformanceProfilePod()
+		Expect(err).ToNot(HaveOccurred(), "Failed to find the PAO pod")
+
+		masterNodes, err := nodes.GetByRole(testutils.RoleMaster)
+		Expect(err).ToNot(HaveOccurred(), "Failed to query the master nodes")
+		for _, node := range masterNodes {
+			if node.Name == pod.Spec.NodeName {
+				return
+			}
+		}
+
+		// Fail
+		Expect(true).To(Reject(), "PAO is not running in a master node")
```

Comment:
change PAO to full name or just "the operator"

Reply:
+1

Reply:
Will do

Reply:
Done